### PR TITLE
changed getpostman link in the docs

### LINKED ISSUE #2580: Old link in Victorops document + scan codebase for links to old domain
At the top of the page content ( https://learning.postman.com/docs/integrations/available-integrations/victorops/ ), we have a link on "Integrations are available on Postman Team, Business, and Enterprise plans." that points to the old domain `getpostman.com`, causing a redirect and dropping key GA data on the floor in doing so.

Update link to point to https://www.postman.com/pricing/ (note trailing slash, otherwise causes another redirect.)

Also, search codebase for any and all `getpostman.com` domains and update to `postman.com` (again, ensure those URLs have trailing slash to fix redirects.)

Screen Shot 2020-08-10 at 8 56 19 AM

### PR DESCRIPTION
**Pull request by akshayysharma**
fix #2580 [links updated]

### CODE REVIEW

**Comment by akshayysharma** [comment]
@SueSmith / @ch264 need you suggestions please.

File: src/pages/docs/integrations/available-integrations/apimatic.md [hunk 1]
```diff
@@ -51,13 +50,13 @@ Click **Authorize** to allow the backup to GitHub.
...
 To back up a collection periodically to a GitHub repository:
 
-* Enter your [API key from APIMatic](https://docs.apimatic.io/getting-started/manage-apis/#view-api-integration-keys).
-* Select a collection.
-* Select a repository.
-* Specify a directory in the repository where you want to add the collection.
-* Enter a filename for the directory in the repository.
-* Select the format in which you want to save the collection, such as Postman 2.0 or APIMatic format.
-* Specify the branch where you want to add the collection. _Note that the branch you want to back up your collection to must already exist._
+- Enter your [API key from APIMatic](https://docs.apimatic.io/getting-started/manage-apis/#view-api-integration-keys).
```
Comment (SueSmith) [inline src/pages/docs/integrations/available-integrations/apimatic.md:53]: this may be causing the lint issue, list items start with * rather than -

Comment (akshayysharma) [reply src/pages/docs/integrations/available-integrations/apimatic.md]: @SueSmith, thankyou for quick help, issue is resolved now.

**Comment by akshayysharma** [comment]
@tristandenyer issue is resolved now and checks is passed. :)

**Comment by SueSmith** [comment]
@akshayysharma make sure you fill out our [contributor form](https://docs.google.com/forms/d/e/1FAIpQLSfbLAcxl-IOiv3NmgEaWw7FleOaXnIyIoIrY_zn6U4JvjQBGA/viewform) so we can send you some cool stuff... 🚀

**Pull request merged**